### DISCUSSION TITLE
onboarded repos not activating

### DISCUSSION THREAD

**Discussion by atloo1** [renovatebot/renovate#32694] [Request Help] [answered]
### How are you running Renovate?

A Mend.io-hosted app

### Please tell us more about your question or problem

I have a cookiecutter templated `.github/renovate.json`. Thus, my workflow has been [1] make repo, `renovate.json` in place, [2] visit https://github.com/apps/renovate, [3] add my repo, [4] get PRs, no onboarding needed. At the onset, both public & private repos had this behavior. https://developer.mend.io/github/atloo1 confirms their `activated` status.  Later repos have been stuck at `onboarded`.

`.github/renovate.json`:
```
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:recommended"
  ],
  "packageRules": [
    {
      "groupName": "patch &/or minor dependency upgrade(s)",
      "matchUpdateTypes": [
        "minor",
        "patch"
      ],
      "prTitle": "patch &/or minor dependency upgrade(s)"
    },
    {
      "matchUpdateTypes": [
        "major"
      ],
      "prTitle": "major dependency upgrade: {{depName}} {{currentVersion}} \u2192 {{newVersion}}"
    }
  ],
  "prConcurrentLimit": 0,
  "prHourlyLimit": 0
}
```

**Comment by atloo1** [comment]
What I believe is happening is that until a PR is proposed, the status can't be `activated`. Can the dev(s) confirm this? A new `poetry` managed repo would be up to date & display the above behavior by this logic.

**Comment by rarkins** [comment, marked as answer]
A repository is "activated" once one or more update PRs have been merged. The only effect this has is how often the Mend app schedules repos behind the scenes. Onboarded repos are scheduled daily, while activated repos get a job every 3-4 hours.
